external/storm-kafka/src/jvm/storm/kafka/SpoutConfig.java

```diff
@@ -29,6 +29,7 @@
 
     // setting for how often to save the current kafka offset to ZooKeeper
     public long stateUpdateIntervalMs = 2000;
+    public String emitStreamId;
```

Comment:
you are right i didnt look at the code too closely, I had a overridden method in PartitionManager with the default stream. I thought anyone else using the old method would automatically go through that. Though thinking now i looked at all the usages and there is only one in the kafkaSpout so i will do another pull request with the changes.